app/config/index.js

```diff
@@ -55,22 +55,26 @@ const schema = Joi.object({
     ...sharedConfigSchema
   },
   applicationRequestMsgType: Joi.string(),
+  applicationResponseMsgType: Joi.string(),
```

Comment:
There are a lot of additions to the config file - has something gone wrong with the merge or are they genuinely required?

Reply:
not sure. It didn't complain.

Reply:
Please remove the additions and edits you've made to this file that aren't related to the test token work. I think you only want the change on line 77 and 143?